### ISSUE TITLE
redocli docker to build html file

### ISSUE THREAD

**Issue by Itarix** [Redocly/redocly-cli#1172]
**Describe the problem to be solved**
For now, the only choice to use redocli is to install via npm/yarn and build with that.

`npm i -g redoc-cli`

`redoc-cli build -o openapi/test.html openapi/test.yml `

**Describe the solution you'd like**

`docker run -it  -v $PWD:/spec redoc-cli build -o output.html swagger.yml`

**Describe alternatives you've considered**
Use npm/yarn install is an alternative. 
But in the CI/CD spirit, we don't want install/build tools each time CI is triggered..

Thanks to taking time.

**Additional context**
Environnement: MacOS Ventura 13.1
CI/CD: GitlabCI.
Docker: Docker version 20.10.20, build 9fdeb9c.

**Comment by tatomyr** [comment]
Hi @Itarix, 

Thanks for reporting! Could you check if this has been resolved in **Redocly CLI** `1.0.0-beta.131`?

cc @dustinblue @knickers

**Comment by knickers** [comment]
I had a successful test 👍

```shell
$ docker pull redocly/cli
$ docker run -it --rm -v "$PWD":/spec redocly/cli --version
1.0.0-beta.131
$ docker run -it --rm -v "$PWD":/spec redocly/cli build-docs redoc-docs/openapi.yaml -o redoc-docs/index.html
Found undefined and using theme.openapi options
Prerendering docs
🎉 bundled successfully in: redoc-docs/index.html (92 KiB) [⏱ 3ms].
$ 
```
